src/west/_bootstrap/main.py

```diff
@@ -136,28 +121,28 @@ def init(argv):
 
 In more detail, does the following:
 
-  1. Clones the manifest repository to west/manifest, and the west repository
-     to west/west
+  1. Clones the manifest repository to a temporary folder, and the west
+     repository to .west/west
 
-  2. Creates a marker file west/{}
+  2. Creates an initial configuration file .west/config
 
-  3. Creates an initial configuration file west/config
+  3. Continues the initialization process through west.main.main()
```

Comment:
Would be better to say "invokes `west init` on the main west installation with an added `--use-cache` parameter"?

Reply:
Well, `west` doesn't invoke `west init` directly, so this wording could lead to confusion in case the reader then expects bootstrapper to execute subprocess.
Bootstrapper actually imports and calls `west.main.main` with an argument list.

But will add some text on `--use-cache`

Reply:
Take a look at updated text.